docs/ops/normalization/MVN_6.md

```diff
@@ -0,0 +1,91 @@
+## MVN <a name="MVN"></a> {#openvino_docs_ops_normalization_MVN_6}
+
+**Versioned name**: *MVN-6*
+
+**Category**: *Normalization*
+
+**Short description**: Calculates mean-variance normalization of the input tensor.
+
+**Detailed description**
+
+*MVN* subtracts mean value from the input blob:
```

Comment:
this example is for case when we normalize dimensions C, H and W (dims 1, 2 and 3) of the 4D input tensor. Please, reflect that in the specification.